kotlin/offline-routing/README.metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "category": "Routing and Logistics",
+    "description": "Solve a route on-the-fly using offline data.",
+    "ignore": false,
+    "images": [
+        "offline-routing.png"
+    ],
+    "keywords": [
+        "connectivity",
+        "disconnected",
+        "fastest",
+        "locator",
+        "navigation",
+        "network analysis",
+        "offline",
+        "routing",
+        "routing",
+        "shortest",
+        "turn-by-turn"
+    ],
+    "redirect_from": "/android/latest/sample-code/offline-routing.htm",
```

Comment:
ah, should we add this to the contribution guide and/or checklist?

Reply:
redirects won't be an issue going forward since all new samples won't need them.

If they're present in the java version of the metadata.json file, they should be copied identically for the kotlin version of the metadata.json file

Reply:
the provision stuff though! I haven't seen this in any metadata yet?